### ISSUE TITLE
How to train models with only random mixes?

### ISSUE THREAD

**Issue by lyndonlauder** [starrytong/SCNet#24]
Hi currently in the config we can set

  remix:
    proba: 1
    group_size: 4

to train with random stems, do you know how to make the training process with only random mixes please?

**Comment by starrytong** [comment]
This is a great idea. I can provide a reference code.

```
class RemixStreams(nn.Module):

    def __init__(self, proba=1):
        super().__init__()
        self.proba = proba

    def forward(self, wav):
        batch, streams, channels, time = wav.size()
        device = wav.device

        if self.training and random.random() < self.proba:
            for idx in range(streams):
                batch_permutation = th.randperm(batch, device=device)
                wav[:, idx, :, :] = wav[batch_permutation, idx, :, :]

        return wav
```
In this code, the streams of different tracks are randomly shuffled.
